demo/locales/source.json

```diff
@@ -443,6 +439,22 @@
     "description": "The header for a section of configuration values.",
     "message": "Language"
   },
+  "DEMO_LCEVC_DRAW_LOGO": {
+    "description": "LCEVC Watermark on the left hand top corner of the canvas.",
+    "message": "Draw LCEVC Logo [JARGON:LCEVC]"
+  },
+  "DEMO_LCEVC_DYNAMIC_PERFORMANCE_SCALING": {
+    "description": "Dynamic Performance scaling for enabling/disabling LCEVC decoding based on decode performance.",
+    "message": "LCEVC Dynamic Performance scaling [JARGON:LCEVC]"
+  },
+  "DEMO_LCEVC_LOG_LEVEL": {
+    "description": "Log Level for LCEVC Lib",
```

Comment:
Please add something like " (0-5)" to the end of this description.  Otherwise it's not clear in the demo UI if this is supposed to be a name like "ERROR" or "DEBUG".